packages/test/test-version-utils/api-report/test-version-utils.api.md

```diff
@@ -16,7 +16,7 @@ import { IFluidDataStoreContext } from '@fluidframework/runtime-definitions';
 import { IFluidDataStoreFactory } from '@fluidframework/runtime-definitions';
 import { IFluidDataStoreRuntime } from '@fluidframework/datastore-definitions';
 import { IFluidLoadable } from '@fluidframework/core-interfaces';
-import * as ink from '@fluidframework/ink';
+import * as ink from '@fluid-experimental/ink';
```

Comment:
since we expose this in test-version-utils, we'll need equivalent back-compat logic in the package loading process to what we did with sequence-deprecated (import from `@fluidframework/ink` if below a specific version, `@fluid-experimental/ink` otherwise).

I'd also be ok with just dropping the coverage and this bit from the test-version-utils API. Your call.

Reply:
I'll elect to drop, we aren't trying to make compat promises about this since it's an experimental dds.